tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -96,6 +96,9 @@ sub run {
     set_var('QESAP_DEPLOYMENT_NAME', get_var('QESAP_DEPLOYMENT_NAME', $deployment_name));
     record_info 'Resource Group', "Resource Group used for deployment: $deployment_name";
 
+    # saptune/mr_test tests use ssh_interactive session which requires the local
+    # ssh config. HanaSR tests do not, so skip creation in non saptune tests
+    #my $provider = $self->provider_factory(skip_ssh_local_config => !get_var('MR_TEST', 0));
```

Comment:
This line is commented here.

Reply:
Yes. Leftover from the version that skips `.ssh/config` creation. Will drop it (and related library changes) if current VRs pass.